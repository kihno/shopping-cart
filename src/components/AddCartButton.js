import React from 'react';

const AddCartButton = (props) => {
    const { handleAdd } = props;

    return (
        <button type="button" className="addButton" onClick={handleAdd}>Add to Cart</button>
    )
}

export default AddCartButton;