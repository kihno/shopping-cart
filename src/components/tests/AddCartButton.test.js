import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import AddCartButton from '../AddCartButton';
import userEvent from '@testing-library/user-event';

describe('Add to cart button', () => {
    it ('renders button', () => {
        const { container } = render(<AddCartButton />);
        expect(container).toMatchSnapshot();
    });
});