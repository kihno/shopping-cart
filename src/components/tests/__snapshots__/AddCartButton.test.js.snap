// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add to cart button renders button 1`] = `
<div>
  <button
    class="addButton"
    type="button"
  >
    Add to Cart
  </button>
</div>
`;
